xml/System.Diagnostics/ActivityLink.xml

```diff
@@ -193,10 +193,10 @@ Activity links can be used to represent batched operations where an activity was
         <Parameter Name="right" Type="System.Diagnostics.ActivityLink" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="left">The first value to compare.</param>
+        <param name="right">The second value to compare.</param>
+        <summary>Returns a value that indicates whether two specified ActivityContext values are not equal.</summary>
+        <returns>true if left and right are not equal; otherwise, false.</returns>
```

Comment:
```suggestion
        <returns>
          <see langword="true" /> if <paramref name="left" /> and <paramref name="right" /> are note equal; otherwise, <see langword="false" />.</returns>
```